### PR TITLE
Simplify delegate invocation.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging   @dotnet/roslyn-ide

**Comment by jasonmalinowski** [comment]
:+1:

**Comment by DustinCampbell** [comment]
:+1:

File: src/Interactive/Features/Interactive/Core/InteractiveHost.cs [hunk 1]
```diff
@@ -151,13 +151,7 @@ private RemoteService TryStartProcess(CultureInfo culture, CancellationToken can
                 newProcess.EnableRaisingEvents = true;
 
                 newProcess.Start();
-
-                // test hook:
```
Comment (Pilchie) [inline src/Interactive/Features/Interactive/Core/InteractiveHost.cs]: You should consider preserving trivia.

**Comment by Pilchie** [comment]
:+1:

File: src/Interactive/Features/Interactive/Core/InteractiveHost.cs (hunk 1 above)
Comment (DustinCampbell) [inline src/Interactive/Features/Interactive/Core/InteractiveHost.cs]: Good catch -- yes!

**Pull request merged**